helm/deploy_test.go

```diff
@@ -132,6 +137,21 @@ func TestHelmDeployConfigureUndeploy(t *testing.T) {
 
 	// Check that the rendered helm env file works
 	validateHelmEnvFile(t, testServiceAccountKubectlOptions)
+
+	// Revoke the tiller service account
+	rbacGroups := []string{}
+	rbacUsers := []string{}
+	serviceAccounts := []string{fmt.Sprintf("%s/%s", namespaceName, testServiceAccountName)}
+	assert.NoError(t, RevokeAccess(kubectlOptions, namespaceName, rbacGroups, rbacUsers, serviceAccounts))
```

Comment:
This should probably be `require.NoError`. We don't want the test to continue if there is an error here.